app/views/user/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if @user.attended_events.any? %>
+	<h2>Upcoming Events</h2> 
+	<ul class="media-list">
+		<%= render @upcoming_events %>
+	</ul>
+<% else %>
+	<h2>No Upcoming Events</h2>
+<% end %>
```

Comment:
- [ ] [Required] Kindly remove this logic from the views, you can always use helper methods for all of the view methods